docs/source/author/content_style_guide.rst

```diff
@@ -0,0 +1,231 @@
+
+The `Unit Plan Template is here`_.
+Sections in italics are descriptions of sections/words that are not are to appear in the final unit plan.
+
+Lessons
+------------------------------------------------------------------------------
+
+The `Lesson Template is here`_.
+Sections in italics are descriptions of sections/words that are not are to appear in the final lesson.
+ 
+Separate folders are used for each age group: 5-7, 8-10, and 11-14.
+Lessons are numbered. 
+ 
+If the same, identical, lesson is used for multiple age groups the file is copied to each folder and a note is added to the top of the document saying “This is identical to <insert link to lesson>”. The original document that is linked to should be the one in the lowest age group, e.g. if the same lesson occurs for 8-10 and 11-14, then the 11-14 copy should contain the note “This is identical to <insert link to lesson>”, rather than the 8-10 lesson.
+ 
+ 
+Learning Outcomes
+------------------------------------------------------------------------------
+
+Each learning outcome has a unique text value, unique key, and belongs to a topic area.
+The list of learning outcomes can be found in the Scorecard and are named according to the rules described in the sections below.
+
+
+Text value
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The text value is the text that will be displayed in the learning outcomes sections on the website.
+These are written using language familiar to teachers and simple enough that it is understandable for students.
+Learning outcomes always begin with a verb.
+
+
+Key
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Learning outcome keys need to follow these rules:
+
+- Words in keys are written in lowercase and are separated with a hyphen e.g. ``explain-different-states``.
+
+- The order of words in the key are:
+  
+  1. The topic area of the outcome
+  2. If the topic area is cross-curricula, the curriculum area of the outcome
+  3. The verb used at the beginning of the text value
+  4. Keywords describing the content of the outcome
+ 
+There is no limit on the length of keys as it is important they are as descriptive as possible.
+
+
+Programming Challenges
+==============================================================================
+
+There needs to be enough scaffolding to support students to be able to achieve a result, independently. 
+
+When transferrring a programming challenge from Google Docs to GitHub, these are the rules to follow:
+
+ 
+1. Separate out all the blocks that “click” together, leaving all the information inside where the parameter is written. All duplicates of a block should be displayed.
+ 
+2. The order of the blocks should always be randomised. If there are more than six blocks the blocks should be split into groups by colour, and then randomise the order of the blocks in these groups. This is the order the colour groups should be displayed in:
```

Comment:
But also if there is six or less blocks, they might as well be split into groups (probably will be automatically as it might be one of each type). This is something I can bring up with @uccser/cs-unplugged-authors, as the rule seems pointless with that threshold.